pkgs/top-level/python-packages.nix

```diff
@@ -2285,6 +2291,24 @@ in {
 
   exrex = callPackage ../development/python-modules/exrex { };
 
+  extractcode = callPackage ../development/python-modules/extractcode { };
+
+  extractcode-7z = callPackage ../development/python-modules/extractcode/7z.nix {
+    inherit (pkgs) p7zip;
```

Comment:
There's no need to do this unless there is a `pythonPackages.p7zip`.

Reply:
I figured it's better to be explicit, in case there ever is a `python*.pkgs.p7zip` that breaks this. Is that undesirable?

Reply:
I guess it's up to the maintainer.